src/couch_server.erl

```diff
@@ -173,6 +174,15 @@ hash_admin_passwords(Persist) ->
             config:set("admins", User, ?b2l(HashedPassword), Persist)
         end, couch_passwords:get_unhashed_admins()).
 
+close_db_if_idle(DbName) ->
+    case ets:lookup(couch_dbs, DbName) of
```

Comment:
I think I'd keep this on the off chance we save sending an extra message to couch_server.